docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
 Write the Docs Worldview
 ------------------------
 
-Our worldview is important for starting a meetup.
-We aren't just another tech writing meetup,
-but hope to be welcoming to the vast parts of the tech industry that care about documentation.
+Yes, Write the Docs has a worldview.
+We include everyone who is interested in excellent documentation: tech writers, developers, engineers, QA, product managers, support engineers and more.
 
-Eric Holscher's Introduction to the conference and community from the Write the Docs 2016 event in Portland is a good place to start.
-We highly recommend watching this before starting a meetup!
+For more information on the Write the Docs worldview, see Eric Holscher's Introduction to the conference and community from Write the Docs NA 2016:
 
-.. note:: You can skip ahead to 1:30, since the beginning is mostly conference logistics.
+.. note:: The "meat" of the worldview starts at 1:30.
 
 .. raw:: html
 
     <iframe width="640" height="360" src="https://www.youtube.com/embed/Fmfs-iYqx5Q?list=PLmV2D6sIiX3U03qc-FPXgLFGFkccCEtfv" frameborder="0" allowfullscreen></iframe>
 
-Done? Let's go into details.
+Now for some details
 
 Launch your Group
 -----------------
 
-Write the Docs meetups are organized using `Meetup <http://www.meetup.com/>`_. Through Meetup, you can create events, contact Meetup members, promote your sponsors, and more. If you need help with Meetup fees, `contact us <mailto:conf@writethedocs.org>`_. We may be able to help.
+You can start a Write the Docs Meetup via `Meetup <http://www.meetup.com/>`_. Through Meetup.com, you can create events, contact Meetup members, promote your sponsors, and more. If you need help with Meetup fees, `contact us <mailto:conf@writethedocs.org>`_. We may be able to help.
 
```

Comment:
You can start a Write the Doc Meetup by using the Meetup.com service. You can use the website or mobile app to  create events, contact meetup members, promote your sponsors and more. If you need help with ....